logical_switch.go

```diff
@@ -331,6 +331,70 @@ func (odbi *ovndb) lsExtIdsDelImp(ls string, external_ids map[string]string) (*O
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
+// lsAuxAddImp is for adding records to external_ids or other_config tables
+func (odbi *ovndb) lsAuxAddImp(ls string, auxConf map[string]string, auxTable string) (*OvnCommand, error) {
+	if (auxTable != "external_ids") && (auxTable != "other_config") {
+		return nil, fmt.Errorf("%s is neither other_config or external_ids", auxTable)
+	}
+
+	var operations []libovsdb.Operation
+	row := make(OVNRow)
+	row["name"] = ls
+	lsuuid := odbi.getRowUUID(TableLogicalSwitch, row)
+	if len(lsuuid) == 0 {
+		return nil, ErrorNotFound
+	}
```

Comment:
Is L343-346, really required? Say, that logical switch is bogus, eventually the commaunt would fail when we execute it later, isn't it? 

Also, this check might succeed now, but little later when the command is run it could be that the logical switch has been deleted.

Reply:
makes sense. will remove